kde/qtcurve/Portfile

```diff
@@ -1,86 +1,241 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; truncate-lines: t; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
-# $Id$
-# kde/libkgapi/Portfile
 
 PortSystem          1.0
-PortGroup           kde4 1.1
 
 fetch.type          git
 git.url             git://anongit.kde.org/qtcurve.git
-git.branch          660f859556471f13834ef4fb095816639aa85014
 
-name                QtCurve
-version             1.8.18
-revision            1
```

Comment:
The current revision of port {{{qtcurve}}} in MacPorts' central repo is 1, i.e. we do need to revbump to 2.